specification/tables/cost_and_usage/purpose.md

```diff
@@ -0,0 +1,2 @@
+The Cost and Usage table is a record of the costs and usage of resources or services in a given billing period. It is used to track and analyze the financial aspects of resource consumption.
```

Comment:
Can we refer to these as "datasets" rather than "tables"?
```suggestion
The Cost and Usage dataset is a record of the costs and usage of resources or services in a given billing period. It is used to track and analyze the financial aspects of resource consumption.
```